steering_modes:
  rover:
    normal:
      linear:
        scale_coefficient: 0.08 # max velocity/effort in range (0,1)
        shape_coefficient: 3.0 # 1.0 - linear characteristics, 2.0 - square-antisymmetric, etc.
      angular:
        scale_coefficient: 0.11
        shape_coefficient: 3.0
    tank:
      scale_coefficient: 0.08
      shape_coefficient: 3.0
    gamer:
      linear:
        scale_coefficient: 0.08
        shape_coefficient: 3.0 
      angular:
        scale_coefficient: 0.8 # in rad, used only in 'gamer' mode
        shape_coefficient: 1.0
  manip:
    normal:
      arm_rotate:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
      arm_lift:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
      claw_rotate:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
      claw_lift:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
      arm_tilt:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
      claw_clamp:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
    gamer:
      arm_rotate:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
      arm_lift:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
      claw_rotate:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
      claw_lift:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
      arm_tilt:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
      claw_clamp:
        scale_coefficient: 1.0
        shape_coefficient: 1.0
    # inverse_kinematics:
    #   x:
    #     scale_coefficient: 1.0
    #     shape_coefficient: 1.0
    #   y:
    #     scale_coefficient: 1.0
    #     shape_coefficient: 1.0
    #   z:
    #     scale_coefficient: 1.0
    #     shape_coefficient: 1.0
    #   theta:
    #     scale_coefficient: 1.0
    #     shape_coefficient: 1.0
    #   fi:
    #     scale_coefficient: 1.0
    #     shape_coefficient: 1.0
    #   claw_clamp:
    #     scale_coefficient: 1.0
    #     shape_coefficient: 1.0
